transifex.yml

```diff
@@ -6,6 +6,16 @@ filters:
     source_file: app/src/main/res/values/strings.xml
     # path expression to translation files, must contain <lang> placeholder
     translation_files_expression: app/src/main/res/values-<lang>/strings.xml
+  - filter_type: file
+    file_format: TXT
+    source_language: en_US
+    source_file: fastlane/metadata/android/en-US/full_description.txt
+    translation_files_expression: fastlane/metadata/android/<lang>/short_description.txt
+  - filter_type: file
+    file_format: TXT
+    source_language: en_US
+    source_file: fastlane/metadata/android/en-US/full_description.txt
```

Comment:
likewise, shouldn't this be  `fastlane/metadata/android/en-US/short_description.txt`?